docs/docs/creating-a-starter.md

```diff
@@ -15,17 +15,19 @@ For a starter to work properly, it needs to include some files (see the [Hello W
 - `static`: a directory for static assets, such as a `favicon.ico` file.
 - `.gitignore`: a file telling Git which resources to leave out of source control, such as the `node_modules` directory, log files, Gatsby `.cache` and `public` directories.
 - `.prettierrc` _(optional)_: a configuration file for [Prettier](https://prettier.io/), a JavaScript linter and formatter used for Gatsby development.
+- `LICENSE`: a file containing an [appropriate](#open-source-license) open source license, preferably with a [BSD Zero Clause License](https://choosealicense.com/licenses/0bsd/).
```

Comment:
```suggestion
- `LICENSE`: a file containing an appropriate [open source license](#open-source-license), preferably with a [BSD Zero Clause License](https://choosealicense.com/licenses/0bsd/).
```

Reply:
I missed that this was navigating to a different section rather than the glossary :facepalm: We should definitely be linking more than the word "appropriate" for accessibility reasons.